tap/tcp_stream_factory.go

```diff
@@ -61,15 +61,14 @@ func (factory *tcpStreamFactory) New(net, transport gopacket.Flow, tcpLayer *lay
 	stream := NewTcpStream(isTapTarget, factory.streamsMap, getPacketOrigin(ac))
 	reassemblyStream := NewTcpReassemblyStream(fmt.Sprintf("%s:%s", net, transport), tcpLayer, fsmOptions, stream)
 	if stream.GetIsTapTarget() {
-		_stream := stream.(*tcpStream)
-		_stream.setId(factory.streamsMap.NextId())
+		stream.setId(factory.streamsMap.NextId())
 		for i, extension := range extensions {
 			reqResMatcher := extension.Dissector.NewResponseRequestMatcher()
 			counterPair := &api.CounterPair{
 				Request:  0,
 				Response: 0,
 			}
-			_stream.addClient(
+			stream.addClient(
```

Comment:
This is only a change in style.
I think we should decide on a convention as a time instead of running over each other's code.

Reply:
What do you mean only change in style?

Before there were both `_stream` and `stream`, one was the concrete struct while the other was the interface.
Now, we only use the concrete struct, as there is no need to use the `interface` internally.

Reply:
Sorry, missed it.